internal/mysql/node.go

```diff
@@ -147,12 +147,32 @@ func (n *Node) getQuery(name string) string {
 
 func (n *Node) traceQuery(query string, arg interface{}, result interface{}, err error) {
 	query = queryOnliner.ReplaceAllString(query, " ")
+	if n.config.ShowOnlyGTIDDiff && IsGtidQuery(query) {
+		n.logger.Debug("<gtid query was ignored>")
+		return
+	}
 	msg := fmt.Sprintf("node %s running query '%s' with args %#v, result: %#v, error: %v", n.host, query, arg, result, err)
 	msg = strings.ReplaceAll(msg, n.config.MySQL.Password, "********")
 	msg = strings.ReplaceAll(msg, n.config.MySQL.ReplicationPassword, "********")
 	n.logger.Debug(msg)
 }
 
+func IsGtidQuery(query string) bool {
+	for _, gtidQuery := range GtidQueries {
+		if strings.HasPrefix(query, gtidQuery) {
+			return true
+		}
+	}
+
+	return false
+}
+
+var GtidQueries = []string{
+	DefaultQueries[queryGTIDExecuted],
+	strings.ReplaceAll(DefaultQueries[querySlaveStatus], ":channel", "''"),
+	strings.ReplaceAll(DefaultQueries[queryReplicaStatus], ":channel", ""),
```

Comment:
Shouldn't this `ReplaceAll`'s arguments too include `"''"`, instead of `""`, similar to the line above?